packages/eslint-plugin/rules/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require( 'requireindex' )( __dirname );
```

Comment:
is it equivalent to re-exporting all files in the folder? just being curious :)

Reply:
It’s equivalent to something like:

```js
module.exports = {
    foo: require( './foo' ),
    bar: require( './bar' ),
    baz: require( './baz' ),
};
```